example/ruby/Gemfile.lock

```diff
@@ -0,0 +1,34 @@
+GEM
```

Comment:
should this be included in PR?

Reply:
No, this is not a library that needs to restrict relevant versions strongly.